docs/output.md

```diff
@@ -0,0 +1,55 @@
+# nf-core/neutronstar: Output
+
+This document describes the output produced by the pipeline. Most of the plots are taken from the MultiQC report, which summarises results at the end of the pipeline.
+
+## Pipeline overview
+The pipeline is built using [Nextflow](https://www.nextflow.io/)
+and processes data using the following steps:
+
+* [Supernova](https://support.10xgenomics.com/de-novo-assembly)
+* [QUAST](http://bioinf.spbau.ru/quast)
+* [BUSCO](http://busco.ezlab.org/)
+* [MultiQC](http://multiqc.info/)
+
+## Supernova
+
+By default the output from Supernova will be stored in two places.
+
+1. ./results/supernova/sample_id. This contains the assembly graph files, log files and statistics from supernova_asm1. See the [documentation](https://support.10xgenomics.com/de-novo-assembly/software/pipelines/latest/output/overview) from 10XG for more details.
+
+2. ./results/assemblies/sample_id[.phased].fasta. These are the assembly fasta files. Two of the four [outputs](https://support.10xgenomics.com/de-novo-assembly/software/pipelines/latest/output/generating) from Supernova are generated:
+    * sample_id.fasta, the `pseudohaploid` version
+    * sample_id.phased.fasta, the `megabubbles` version where haplotypes are included as two separate fasta entries.
+
+To generate the two other output types, for instance `raw` you can use the command `supernova mkoutput --asmdir=./results/supernova/sample_id/outs/assembly --style=raw --outprefix=sample_id.raw`
```

Comment:
Code block breaks in the github suggestion, but I think longer commands like this should be in a block rather inline:

---
To generate the two other output types, for instance `raw` you can use the following command:
```bash
supernova mkoutput --asmdir=./results/supernova/sample_id/outs/assembly --style=raw --outprefix=sample_id.raw
```